force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -126,16 +150,28 @@ public with sharing class ClassesObjectsMethods {
         List<String> scientistNames = new List<String>{'Tim Berners-Lee', 'Alan Turing', 'Grace Hopper', 'Donald Knuth', 'Guido van Rossum', 'Ken Thompson', 'Stephen Hawking'};
 
         // The variable to store the concatenated string
+        String filteredNames = '';
 
         // Loop through the list of scientists' names
-
+        for (String name : scientistNames) {
             // If the name doesn't contain the letter 'G', skip this iteration
-
-            // Add the name to the result string, followed by a comma
-
+            if (name.containsIgnoreCase('G')) {
+                filteredNames += name + ',';
+            } else {
+                continue;
+            }
+
+            // if (name.toLowerCase().containsNone('g')) {
+            //     continue;
+            // } else {
+            //     // Add the name to the result string, followed by a comma
+            //     filteredNames += name + ',';
+            // }
```

Comment:
To clean up your code, be sure to delete blocks of commented-out code you have no intention of using.

Reply:
Normally I would, I had left those for reference for some reason, but agree. Thanks.